src/api/docs/content/specs/queries.yaml

```diff
@@ -114,6 +114,7 @@ components:
             description: Load queries from on-disk database rather than from in-memory
             required: false
             schema:
+              type: boolean
               enum:
                 - true
                 - false
```

Comment:
Here the type is `enum` (enumerated type or enumerated set), containing 2 items: `true` and `false`.